src/exampleTypes/Geo.js

```diff
@@ -71,10 +72,11 @@ let GeoComponent = injectTreeNode(
                 const inputValue = newValue.replace(/[^a-zA-Z0-9\s]+/g, '')
                 return inputValue
               }}
-              onChange={async ({ value }) => {
-                let data = await GeoCodeLocation(value)
+              onChange={async selectedOption => {
+                let data = await GeoCodeLocation(selectedOption.value)
                 if (data && data.latitude && data.longitude && data.location) {
                   tree.mutate(node.path, {
+                    selectedOption,
```

Comment:
As a general rule, we should not add component props to contexture nodes. Contexture nodes do not necessarily map one to one with react components. In practice, they often do - but that's not guaranteed to be the case. 

Also, this would result in saving "label" in the database, which isn't right. Instead, we can just have the value be the first option that matches latitude and longitude - no need to persist this intermediate state